src/test/java/ch/uzh/ifi/hase/soprafs22/service/UserServiceIntegrationTest.java

```diff
@@ -59,6 +61,7 @@ void createUser_validInputs_success() {
     testUser.setStatus(UserStatus.OFFLINE);
     testUser.setToken("a");
     testUser.setPassword("1234");
+    testUser.setBirthday(new Date());
```

Comment:
Why do you set the birthday as today (would make more sense to have him be older than 18)